.github/workflows/README.md

```diff
@@ -0,0 +1,3 @@
+Check out the guide that was used to create the CI environment including setting up the yaml files:
+
+https://medium.com/@cjolowicz/hypermodern-python-d44485d9d769
```

Comment:
Let's use the link to https://cjolowicz.github.io/posts/hypermodern-python-01-setup/. Not everything can read Medium these days.

Reply:
Will amend through #95.